spec/models/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    before(:example) { get user_posts_path(1) }
+
+    it 'should have correct response status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should include correct placeholder text' do
+      expect(response.body).to include('Posts#index')
+    end
+  end
+
+  describe 'GET #show' do
+    before(:example) { get user_post_path(1, 1) }
+
+    it 'should have correct response status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'should include correct placeholder text' do
+      expect(response.body).to include('Posts#show')
+    end
+  end
```

Comment:
- [ ] This should make your tests not run because we are supposed to target the text that has been included in the body. however, you included ('Posts#index).

HINT: expect(response.body).to include("Welcome to user index page").